### PR TITLE
Login gov

### PR DESCRIPTION
**Pull request by thejuliekramer**

### CODE REVIEW

File: ckanext/saml2/plugin.py [hunk 1]
```diff
@@ -304,21 +304,28 @@ def identify(self):
...
+            
         except KeyError:
             # This is a request in an existing session so no need to provision
             # an account, set c.userobj and return
             c.userobj = model.User.get(c.user)
             if c.userobj is not None:
-                c.user = saml_info['maxemail'][0]
+                c.user = saml_info['maxemail'][0] # REMIND: this looks suspect
+
+            log.debug("identify(): KeyError; saml_info = %r", saml_info)
```
Comment (thejuliekramer) [inline ckanext/saml2/plugin.py:327]:
```suggestion
           
```

File: ckanext/saml2/plugin.py [hunk 2]
```diff
@@ -304,21 +304,28 @@ def identify(self):
...
             c.user = saml2_user_info[1].name
 
+        log.debug("identify(): c.user = %r", c.user)
         log.debug("repoze.who.identity = {0}".format(dict(environ["repoze.who.identity"])))
 
         # get the actual user info from the saml2auth client
         try:
             saml_info = environ["repoze.who.identity"]["user"]
-
+            log.debug("identify(): saml_info = %r", saml_info)
```
Comment (thejuliekramer) [inline ckanext/saml2/plugin.py:318]:
```suggestion

```

File: ckanext/saml2/plugin.py [hunk 3]
```diff
@@ -304,21 +304,28 @@ def identify(self):
         log.debug('NameId: %s' % (name_id))
 
         saml2_user_info = saml2_get_user_info(name_id)
+        log.debug("saml2_user_info = %r", saml2_user_info)
+        
         if saml2_user_info is not None:
             c.user = saml2_user_info[1].name
 
+        log.debug("identify(): c.user = %r", c.user)
```
Comment (thejuliekramer) [inline ckanext/saml2/plugin.py:312]:
```suggestion

```

File: ckanext/saml2/plugin.py [hunk 4]
```diff
@@ -304,21 +304,28 @@ def identify(self):
         log.debug('NameId: %s' % (name_id))
 
         saml2_user_info = saml2_get_user_info(name_id)
+        log.debug("saml2_user_info = %r", saml2_user_info)
```
Comment (thejuliekramer) [inline ckanext/saml2/plugin.py:307]:
```suggestion

```

**Review by woodt — APPROVED** [review]
This looks good to me.  We have questions about whether this change would impact catalog.

**Pull request merged**